index.js

```diff
@@ -1,60 +1,69 @@
 'use strict'
 
 const { Client } = require('discord-rpc');
+const julian = require('julian');
+const moment = require('moment');
 
-const clientId = '811111315988283413';
-
+const clientId = '811145575134658561';
 const day = 24 * 60 * 60;
 const landingTimestamp = 1613681692;
 
+function getMSD(earthTime) {
+    earthTime = moment(earthTime).toDate();
+    return (julian(earthTime) - 2405522.0028779) / 1.0274912517;
+};
+function getMissionSol() {
+    let date = new Date().getTime();
+
+    return Math.floor(getMSD(date) - getMSD(new Date(1613681692 * 1000).toISOString()));
+}
+
+console.log(new Date(1613681692 * 1000).toISOString());
+
+console.log(getMissionSol());
+
 const rpcClient = new Client({ transport: 'ipc' });
 
 rpcClient.on('ready', () => {
-  update();
-  setInterval(update, 10 * 1000);
+    update();
+    setInterval(update, 10 * 1000);
 });
 
 rpcClient.on('disconnected', () => {
-  console.log('Disconnected, attempting reconnect every 10 seconds');
-  setTimeout(connect, 10000);
+    console.log('Disconnected, attempting reconnect every 10 seconds');
+    setTimeout(connect, 10000);
 })
 
 function update() {
-  const currentTimestamp = Math.round(Date.now() / 1000);
-  const diff = landingTimestamp - currentTimestamp;
-  const diffDays = Math.floor(diff / day);
-  let state = undefined;
-  let details = diff > 0 ? "#CountdownToMars" : 'Perseverance is on Mars!';
-  if (diffDays > 0) {
-    state = `${diffDays} Day${diffDays === 1 ? '' : 's'} and`;
-  }
-  const watchable = diff < 4 * 60 * 60;
-  const buttons = [
-    {
-      label: watchable ? 'Watch the Landing!' : 'How Landing Works!',
-      url: watchable ? 'https://youtu.be/gm0b_ijaYMQ' : 'https://mars.nasa.gov/mars2020/timeline/landing/entry-descent-landing/',
-    },
-    {
-      label: 'Read about Perseverance!',
-      url: 'https://mars.nasa.gov/mars2020/',
-    },
-  ];
-
-  rpcClient.setActivity({
-    state,
-    details,
-    endTimestamp: diff > 0 ? landingTimestamp : undefined,
-    largeImageKey: 'nasa',
-    largeImageText: 'Nasa is running this mission!',
-    buttons,
-  }).catch(console.error);
+    const currentTimestamp = Math.round(Date.now() / 1000);
+    let state = `Perseverance - Sol ${getMissionSol()}`;
+    let details = `Location: Jezero Crater`;
```

Comment:
```suggestion
  const state = `Perseverance - Sol ${getMissionSol()}`;
  const details = `Location: Jezero Crater`;
```
clean up these since we're not updating anymore